src/grpc/helium_packet_router_service.erl

```diff
@@ -0,0 +1,45 @@
+-module(helium_packet_router_service).
```

Comment:
noting it here for reference, this handler was one of the first grpc handlers added and its approach  with the 'receive' loop is maybe a little raw.

The newer handlers implement handle_info msgs and as such are a somewhat more of an otp style approach.  An example of those newer handlers can be seen here:

https://github.com/helium/blockchain-core/blob/master/src/grpc/blockchain_grpc_sc_server_handler.erl

This router handler is obviously a lot simpler and like i say nothing wrong with it on its own but maybe be better to follow the newer pattens of the later handlers especially if this is going to be the basis of other handlers

Reply:
ok I think we will be able to use this, this is just boilerplate to start including GRPCwe actually won't even use this service in there